librz/core/cfile.c

```diff
@@ -492,19 +492,8 @@ RZ_API bool rz_core_file_resize_delta(RzCore *core, st64 delta) {
 	return file_resize(core, 0, delta);
 }
 
-RZ_API void rz_core_sysenv_end(RzCore *core, const char *cmd) {
-	// TODO: remove tmpfilez
-	if (strstr(cmd, "RZ_BLOCK")) {
-		// remove temporary BLOCK file
-		char *f = rz_sys_getenv("RZ_BLOCK");
-		if (f) {
-			rz_file_rm(f);
-			rz_sys_setenv("RZ_BLOCK", NULL);
-			free(f);
-		}
-	}
+RZ_API void rz_core_sysenv_end(RzCore *core) {
```

Comment:
Since you modify this function anyway, could you please add a Doxygen description here?

Reply:
dropping support for this.